components/Card/CardTitle.tsx

```diff
@@ -27,93 +13,151 @@ interface CardTitleProps {
+              </a>{" "}
+              {subheader && (
+                <>
+                  {t("newsfeed.action_update")}
+                  {subheader}
+                </>
+              )}
+            </CardBootstrap.Title>
+          )}
+        </>
+      )
+    default:
+      return (
+        <CardBootstrap.Title
+          className={`align-items-start fs-6 lh-sm mb-1 text-secondary`}
+        >
+          <strong>{header}</strong>
+        </CardBootstrap.Title>
+      )
+  }
+}
+
+const CardTitleFollowing = (props: CardTitleProps) => {
+  const { header, subheader, isBillMatch, isUserMatch, type } = props
+  const { t } = useTranslation("common")
+
+  if (type == `no results`) {
+    return <></>
+  } else if (type === `bill`) {
+    return (
+      <>
         {header && (
-          <CardBootstrap.Title
-            className={`align-items-start fs-6 lh-sm mb-1 text-secondary`}
-          >
-            <a href={`/bills/${court}/${header}`}>
-              <strong>{formatBillId(header)}</strong>
-            </a>{" "}
-            {subheader ? (
-              <>
-                {t("newsfeed.action_update")}
-                {subheader}
-              </>
-            ) : (
-              <></>
-            )}
-          </CardBootstrap.Title>
-        )}
-        {header ? (
           <CardBootstrap.Title
             className={`align-items-start fs-6 lh-sm mb-1 text-body-tertiary`}
           >
-            {header && isBillMatch ? (
+            {isBillMatch ? (
               <>{t("newsfeed.follow")}</>
             ) : (
               <>{t("newsfeed.not_follow")}</>
             )}
             <strong>{formatBillId(header)}</strong>
           </CardBootstrap.Title>
-        ) : (
-          <></>
         )}
-      </CardBootstrap.Body>
-    </CardBootstrap.Body>
-  )
+      </>
+    )
+  } else if (isBillMatch === true && isUserMatch === true) {
+    return (
+      <CardBootstrap.Title
+        className={`align-items-start fs-6 lh-sm mb-1 text-body-tertiary`}
+      >
+        {t("newsfeed.follow")}
+        {header && <strong>{formatBillId(header)}</strong>}
+        {t("newsfeed.and")}
+        {subheader}
+      </CardBootstrap.Title>
+    )
+  } else if (isBillMatch === true && isUserMatch === false) {
+    return (
+      <CardBootstrap.Title
+        className={`align-items-start fs-6 lh-sm mb-1 text-body-tertiary`}
+      >
+        {t("newsfeed.follow")}
+        {header && <strong>{formatBillId(header)}</strong>}
+      </CardBootstrap.Title>
+    )
+  } else if (isBillMatch === false && isUserMatch === true) {
+    return (
+      <CardBootstrap.Title
+        className={`align-items-start fs-6 lh-sm mb-1 text-body-tertiary`}
+      >
+        {t("newsfeed.follow")}
+        {subheader}
+      </CardBootstrap.Title>
+    )
+  } else {
+    return (
+      <CardBootstrap.Title
+        className={`align-items-start fs-6 lh-sm mb-1 text-body-tertiary`}
+      >
+        {t("newsfeed.not_follow_either")}
```

Comment:
Is this a state that can occur? Presumably we'd need to follow either the bill or the user to be notified at all.

Reply:
it's related to an issue I need to submit where after an item has been added to one's newsfeed, your following status as displayed by the newsfeed doesn't update if you subsequently go back and change your actual following status.